docker_test.go

```diff
@@ -1382,6 +1383,26 @@ func TestContainerNonExistentImage(t *testing.T) {
 	})
 }
 
+func TestContainerCustomPlatformImage(t *testing.T) {
+	t.Run("Use a custom image platform", func(t *testing.T) {
```

Comment:
Wdyt about renaming the test case? Because `windows/arm12` is not a valid platform, I guess that even if running the tests on a Windows ARM machine, this will fail, right? 
```suggestion
	t.Run("Use an invalid custom image platform", func(t *testing.T) {
```

Reply:
Hi @mdelapenya this was my not so optimal attempt to test the functionality i am adding. I used `windows/arm12` since that's a non-existent image platform so I expect the test to errror out.

Reply:
wdyt about `Use a non-existent custom image platform` then?

My intention here is to make the test super explicit to casual readers, specially on failures